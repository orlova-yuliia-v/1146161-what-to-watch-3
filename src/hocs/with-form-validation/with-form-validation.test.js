import React from "react";
import renderer from "react-test-renderer";
import withFormValidation from "./with-form-validation.jsx";

const MockComponent = () => <div />;

const MockComponentWrapped = withFormValidation(MockComponent);

it(`should render withFormValidation HOC correctly`, () => {
  const tree = renderer
  .create(
      <
        MockComponentWrapped
        id={0}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
