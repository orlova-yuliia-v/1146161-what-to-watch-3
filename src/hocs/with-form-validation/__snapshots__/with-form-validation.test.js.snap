// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render withFormValidation HOC correctly 1`] = `<div />`;
